adapters/incluster/v1/client.go

```diff
@@ -450,7 +450,11 @@ func (c *Client) PutObject(_ context.Context, id domain.KindName, object []byte)
 	}
 	// use apply to create or update object, we want to overwrite existing objects
 	// TODO for the moment we keep the dynamic client as we create fewer objects than we fetch
-	_, err = c.dynamicClient.Resource(c.res).Namespace(id.Namespace).Apply(context.Background(), id.Name, &obj, metav1.ApplyOptions{FieldManager: "application/apply-patch"})
+	options := metav1.ApplyOptions{
+		FieldManager: "application/apply-patch",
+		Force:        true,
```

Comment:
@amirmalka I think I ran into the issue that we saw last month - do you think it's OK to force here?

Reply:
Eventually we discovered that the problem was trying to apply a namespaced object for a non-namespaced resource definition.
Why do we need to force? What is the issue you are seeing?

Reply:
right... well in this case we really need it since it's an existing object, that was updated in both sides, but we want the backend to have the last word on it
I think mechanism 1 applies here: https://pkg.go.dev/k8s.io/client-go/applyconfigurations#hdr-Controller_Support